### ISSUE TITLE
posting images is unusable, - resized image is too small

### ISSUE THREAD

**Issue by DJRitmas** [mumble-voip/mumble#4830]
Please remove hard coded image resizing in chat. Current value is to 480x270 which is unusable, since there will be server-side configuration that allows restricting image size.

![mumbleImg](https://user-images.githubusercontent.com/80097194/110095272-686d4700-7da5-11eb-87ba-842cfad0cf04.png)

Additionally might need to check max-width/height options in chat display.

**Comment by Gert-dev** [comment]
I tried messing with the code a bit, because I thought a good stopgap might be to size the image to the window size (resizing the window doesn't happen that often and this edge case could be done at a later time), but the sender apparently determines the size for _everyone_ else, so you can't have an ideal size for everyone right now.

Instead, since 1920x1080 is a very common desktop resolution by now, I'm wondering if we can't replace the current code withe something like this as a stopgap measure for now:

```cpp
img = img.scaled(qMin(800, img.width()), qMin(600, img.height()), Qt::KeepAspectRatio, Qt::SmoothTransformation);
```

For most images, this already results in far better legibility and image displays, especially when people post comics or memes in Mumble - you currently need to squint at the screen often, because it's always sized down to 480x270.

Do the Mumble developers feel that this change is acceptable right now as stopgap? I'd be willing to make a PR with it if so.

**Comment by Krzmbrzl** [comment]
This has been resolved (at least to some extend) via #5029
